packages/main/src/plugin/index.ts

```diff
@@ -1607,16 +1607,16 @@ export class PluginSystem {
       },
     );
 
-    this.ipcHandle('proxy:setState', async (_listener: Electron.IpcMainInvokeEvent, state: boolean): Promise<void> => {
-      return proxy.setState(state);
+    this.ipcHandle('proxy:setState', async (_listener: Electron.IpcMainInvokeEvent, state: number): Promise<void> => {
```

Comment:
I wonder why don't you use ProxyState instead of number?

Reply:
```suggestion
    this.ipcHandle('proxy:setState', async (_listener: Electron.IpcMainInvokeEvent, state: ProxyState): Promise<void> => {
```